chromium_src/chrome/browser/ui/ui_features.cc

```diff
@@ -11,6 +11,9 @@
 namespace features {
 
 OVERRIDE_FEATURE_DEFAULT_STATES({{
+#if !defined(ANDROID)
```

Comment:
`!BUILDFLAG(IS_ANDROID)`

Reply:
For some reason they used `defined` upstream: https://source.chromium.org/chromium/chromium/src/+/refs/tags/113.0.5672.24:chrome/browser/ui/ui_features.cc;l=76

Reply:
Changed in b3485429e99